lib/utils/isStandardSyntaxRule.js

```diff
@@ -37,6 +37,11 @@ module.exports = function (rule/*: Object*/)/*: boolean*/ {
     return false
   }
 
+  // Less guards
+  if ((/when.*?\(.*?\)/).test(selector)) {
```

Comment:
@evilebottnawi: You can make the regexp as complicated as you think necessary — but I think it's not great to make it so simple that it might raise false positives.